src/features/register/components/registration-form/index.tsx

```diff
@@ -43,96 +48,111 @@ const RegistrationForm = () => {
+              message: 'Please input your E-mail!',
+            },
+          ]}
+        >
+          <Input />
+        </Form.Item>
 
-      <Form.Item
-        name="password"
-        label="Password"
-        rules={[
-          {
-            required: true,
-            message: 'Please input your password!',
-          },
-        ]}
-        hasFeedback
-      >
-        <Input.Password />
-      </Form.Item>
+        <Form.Item
+          name="password"
+          label="Password"
+          rules={[
+            {
+              required: true,
+              message: 'Please input your password!',
+            },
+          ]}
+          hasFeedback
+        >
+          <Input.Password />
+        </Form.Item>
 
-      <Form.Item
-        name="confirm"
-        label="Confirm Password"
-        dependencies={['password']}
-        hasFeedback
-        rules={[
-          {
-            required: true,
-            message: 'Please confirm your password!',
-          },
-          ({ getFieldValue }) => ({
-            validator(_, value) {
-              if (!value || getFieldValue('password') === value) {
-                return Promise.resolve();
-              }
-              return Promise.reject(
-                new Error('The two passwords that you entered do not match!'),
-              );
+        <Form.Item
+          name="confirm"
+          label="Confirm Password"
+          dependencies={['password']}
+          hasFeedback
+          rules={[
+            {
+              required: true,
+              message: 'Please confirm your password!',
             },
-          }),
-        ]}
-      >
-        <Input.Password />
-      </Form.Item>
+            ({ getFieldValue }) => ({
+              validator(_, value) {
+                if (!value || getFieldValue('password') === value) {
+                  return Promise.resolve();
+                }
+                return Promise.reject(
+                  new Error('The two passwords that you entered do not match!'),
+                );
+              },
+            }),
 
-      <Form.Item
-        name="displayName"
-        label={
-          <span>
-            Display Name&nbsp;
-            <Tooltip title="What do you want others to call you?">
-              <QuestionCircleOutlined />
-            </Tooltip>
-          </span>
-        }
-        rules={[
-          {
-            required: true,
-            message: 'Please input your display name!',
-            whitespace: true,
-          },
-        ]}
-      >
-        <Input />
-      </Form.Item>
+            ({ getFieldValue }) => ({
+              validator(_, value) {
+                if (value.length >= 8 || getFieldValue('password') >= 8) {
+                  return Promise.resolve();
+                }
+                return Promise.reject(
+                  new Error('Must have at least 8 characters!'),
+                );
+              },
+            }),
+          ]}
```

Comment:
```suggestion
```

I believe the check should happen at the Password field not Confirm Password field 🤔 An user won't want to have typed their incorrect password twice to then realise they need to change it.